arangod/Graph/GraphOperations.h

```diff
@@ -106,6 +106,27 @@ class GraphOperations {
   OperationResult createEdge(const std::string& definitionName, VPackSlice document,
                              bool waitForSync, bool returnNew);
 
+  // @brief This function is a helper function which is setting up a transaction
+  // and calls validateEdgeVertices and validateEdgeContent methods.
+  std::pair<OperationResult, std::unique_ptr<transaction::Methods>> validateEdge(
+      const std::string& definitionName, const VPackSlice& document,
+      bool waitForSync, bool isUpdate);
+
+  // @brief This function is checking whether the given _from and _to vertex documents are available or not
+  OperationResult validateEdgeVertices(const std::string& fromCollectionName,
+                                       const std::string& fromCollectionKey,
+                                       const std::string& toCollectionName,
+                                       const std::string& toCollectionKey,
+                                       transaction::Methods& trx);
+
+  // @brief This function is checking whether the given document defines _from and _to attributes or not
+  // and checks if they are correct or invalid if they are available.
```

Comment:
What constitutes correctness or validity of the _from and _to attributes?

Reply:
We're checking if, 
1.) They are available (both, from and to)
2.) If they conform to "from and to" syntax. (Yes, we should use or create a global function here in the future)
3.) Then it will be checked if they are available inside the graph definitions